### ISSUE TITLE
Product Editor: Variable inventory settings differ from classic experience

### ISSUE THREAD

**Issue by mattsherman** [woothemes/woocommerce#48952]
**Issue Description:**

In the classic experience, the merchant can track stock quantity for a variable product, and set options that apply to all variations without manual stock management enabled.

<img width="817" alt="Screenshot 2024-06-28 at 12 48 33" src="https://github.com/woocommerce/woocommerce/assets/2098816/496592ef-9674-4d26-a4c5-fa940d4ec630">

In the new product editor, it is not possible to set these options at the variable product level.

<img width="672" alt="Screenshot 2024-06-28 at 12 49 25" src="https://github.com/woocommerce/woocommerce/assets/2098816/b3e279c5-45ac-468f-ad23-5c59fe1e8159">

This seems like it will be a backwards compatibility issue and is something we should look into to verify whether this is actually a breaking change we want to make, or whether we should be compatible with the classic experience.

**Comment by j111q** [comment]
> This seems like it will be a backwards compatibility issue and is something we should look into to verify whether this is actually a breaking change we want to make, or whether we should be compatible with the classic experience.

Yes, I can guess why this was initially scoped out (focusing on driving individual variation settings instead of bulk settings) but agree we should support backwards compatibility here. 

Here's a mock-up that I did, I would suggest we add the guidance about the stock setting as a help text underneath the toggle:

Toggled Off:

![image](https://github.com/woocommerce/woocommerce/assets/65582817/410dbb6b-c293-4842-8d03-fbeaca960fa3)

Toggled On:

<img width="1297" alt="image" src="https://github.com/woocommerce/woocommerce/assets/65582817/3b1c9e97-ac37-4300-bc21-63fab74da413">

Does that work?

**Comment by mattsherman** [comment]
@j111q I think that approach works -- thanks!

**Comment by gigitux** [comment]
Since the release of the [What's Next in the Product Editor](https://developer.woocommerce.com/2024/11/06/whats-next-in-the-product-editor/) announcement, this issue is no longer relevant due to changes in the Product Editor's direction.  

If you find that this is still a concern with the latest updates, please provide additional context, or feel free to reopen the issue.  

Thank you for your understanding and contributions!
